store/projectVersion.js

```diff
@@ -0,0 +1,52 @@
+import { create } from "zustand";
+
+const useProjectVersionStore = create(set => {
+  return {
+    versionInformation: {
+      byDates: {},
+      allDates: [],
+    },
+    setVersion: versionList => {
+      versionList.forEach(version => {
+        const { id, created_at: createdAt, label } = version;
+
+        const createdDate = createdAt.slice(0, 10);
+        const createdTime = createdAt.slice(11, 16);
```

Comment:
P3: `createdDate`, `createdTime`이 `createdAt`이라는 데이터 형식인 "YYYY-MM-DDTHH:MM:SS"을 분리한
데이터라는 게 직관적으로 보이면 좋을 것 같습니다. 아래 방식처럼 객체분해할당을 활용해서 한 줄로 코드를 사용하는 건 어떨까요? 

```suggestion
const [ createdDate, createdTime ] = createdAt.split("T").slice(0, 5);
```